lib/features/guide_detail_view/widgets/my_expansion_tile.dart

```diff
@@ -0,0 +1,71 @@
+import "package:flutter/material.dart";
+
+import "../../../config/ui_config.dart";
+import "../../../theme/app_theme.dart";
+import "../../../widgets/loading_widgets/shimmer_loading.dart";
+
+class MyExpansionTile extends StatelessWidget {
+  final String title;
+  final String description;
+
+  const MyExpansionTile({
+    required this.title,
+    required this.description,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        horizontal: GuideDetailViewConfig.paddingMedium,
+      ),
+      child: ExpansionTile(
```

Comment:
Yes, I think it's very good you've used this built in one. Let's stick with it I think